blueocean-rest-impl/pom.xml

```diff
@@ -51,14 +51,14 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-credentials</artifactId>
         </dependency>
-        <dependency> <!-- need this to solve a JSch linkage problem -->
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>git-client</artifactId>
-        </dependency>
         <dependency>
             <groupId>oro</groupId>
             <artifactId>oro</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+        </dependency>
```

Comment:
OK, you are bundling this in your `*.hpi` and not sharing it or anything.

Reply:
Use the plugin.

Reply:
Cf. https://github.com/jenkinsci/ssh-credentials-plugin/pull/19

Reply:
Fixed, depends on jsch plugin.